README.adoc

```diff
@@ -480,6 +480,47 @@ _ { raise_exception }.must_raise TypeError
 
 Check the http://docs.seattlerb.org/minitest/Minitest/Expectations.html[Minitest::Expectations doc] for more information about its usage.
 
+=== Hooks [[hooks]]
+
+If using a module containing `setup` or `teardown` methods, be sure to call `super` in the test class `setup` or
+`teardown`.
+
+[source,ruby]
+----
+# bad
+module MyHelper
+  def setup
+    stub_something
+  end
+end
```

Comment:
The bad case and the good case are the same. For POROs that are not inherited from `Minitest::Test`, it may be clearer to remove them from the example code.

Reply:
I've removed it.